aiken-contracts/fractional/scripts/staking/04_deregisterStake.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
I'm assuming this was an attempt to deregister stake key but it didn't end up working?

Reply:
Yeah, this file should always fail.